test/lumo/repl_tests.cljs

```diff
@@ -61,7 +61,9 @@
                                              '{string clojure.string})]
         (is-contains-completion "str" "string/")
         (is-contains-completion "(str" "(string/")
-        (is-contains-completion "(set" "(set/" not))
+        (is-contains-completion "(set" "(set/" not)
+        (testing "LUMO-258"
+          (is-contains-completion "string/ends" "string/ends-with?")))
```

Comment:
This is not enough. We also want to make sure that `ends_with_QMARK_` isn't there. That's the problem

Reply:
Ok